kxstudio/cardinal.spec

```diff
@@ -116,6 +119,20 @@ EOF
 install -m 755 -d %{buildroot}/%{_datadir}/icons/hicolor/scalable/apps/
 install -m 644 %{SOURCE1} %{buildroot}/%{_datadir}/icons/hicolor/scalable/apps/
 
+# Remove empty file
+rm -- %{buildroot}%{_datadir}/%{name}/surgext/patches/README.md
```

Comment:
What is the use of '--' with rm ? is it equivalent to rm -f ?

Reply:
-- separate options from file arguments. Just habit, I can remove since it doesn't do anything here